### PR TITLE
Added chargers

### PR DESCRIPTION
**Pull request by wvanderp**

### CODE REVIEW

File: data/operators/amenity/charging_station.json [hunk 1]
```diff
@@ -3664,13 +3665,24 @@
         "operator": "Parkering Göteborg"
       }
     },
+    {
+      "displayName": "City Charging",
+      "locationSet": {"include": ["001"]},
```
Comment (kjonosm) [inline data/operators/amenity/charging_station.json:3670]: locationSet should be limited to `"nl"`

File: data/operators/amenity/charging_station.json [hunk 2]
```diff
@@ -3664,13 +3665,24 @@
...
+      "tags": {
+        "amenity": "charging_station",
+        "operator": "City Charging",
+        "operator:wikidata": "Q126405534"
+      }
+    },
     {
       "displayName": "PARKnCHARGE",
```
Comment (kjonosm) [inline data/operators/amenity/charging_station.json:3678]: According to their website they are branded as `Park&Charge`

File: data/operators/amenity/charging_station.json [hunk 3]
```diff
@@ -3664,13 +3665,24 @@
...
       "id": "parkncharge-c22d3a",
       "locationSet": {"include": ["001"]},
+      "matchNames": ["Park n Charge", "Park 'n Charge", "Park&Charge"],
       "tags": {
         "amenity": "charging_station",
-        "operator": "PARKnCHARGE"
+        "operator": "PARKnCHARGE",
+        "operator:wikidata": "Q109733858"
```
Comment (kjonosm) [inline data/operators/amenity/charging_station.json:3685]: This looks like a wrong Wikidata QID to be replaced by `Q126404473`

**Comment by wvanderp** [comment]
Thank you for the review. and my apologies for making a preventable mistake.

I've now checked my work and applied the changes and the pr is ready for rereview.

**Comment by bhousel** [comment]
thanks for the fixes @wvanderp  👍

**Pull request merged**